custom-recipes/timeseries-preparation-resampling/recipe.json

```diff
@@ -85,7 +85,8 @@
       "name": "time_step",
       "label": "Time step",
       "type": "DOUBLE",
-      "defaultValue": 1
+      "defaultValue": 1,
+      "minI": 1
```

Comment:
"minI" is not working because the parameter type is "DOUBLE".
Do you know if it's "DOUBLE" on purpose and we should use "minD" or if it should be "INT" ?
From my tests, doubles fail only on weeks, months and year frequencies.
If we keep DOUBLE, then minD should be 0 instead on 1 (even though 0 will fail)

Reply:
Because of this [code snippet](https://github.com/dataiku/dss-plugin-timeseries-preparation/blob/dbdc349d9ae4bfdddacbf43618d57303fe5a019b/python-lib/dku_timeseries/timeseries_helpers.py#L44), I think "DOUBLE" is set on purpose yet it works only with these frequencies : `ROUND_COMPATIBLE_TIME_UNIT = ['days', 'hours', 'minutes', 'seconds', 'milliseconds', 'microseconds', 'nanoseconds']`

Reply:
replaced with 0 in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/23/commits/7eb609fbb039658f4a7a4371acc15208a6a5cc9a

Reply:
also it needs to be "minD" instead of "minI"

Reply:
done in https://github.com/dataiku/dss-plugin-timeseries-preparation/pull/23/commits/40c793769c6f54a68999e8e2195f0bcc0208b9e3